Compilation problem gcc 8.2.1

I solved it with [[maybe_unused]] but I don't know yet if this is good solution.
```
Scanning dependencies of target frozen.tests
[  5%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_algorithms.cpp.o
[ 10%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_main.cpp.o
[ 15%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_map.cpp.o
[ 20%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_rand.cpp.o
[ 25%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_set.cpp.o
[ 30%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_str.cpp.o
[ 35%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_str_set.cpp.o
[ 40%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_unordered_map.cpp.o
[ 45%] Building CXX object tests/CMakeFiles/frozen.tests.dir/test_unordered_map_str.cpp.o
/home/capsel/libs/frozen/tests/test_unordered_map_str.cpp: In function ‘void ____C_A_T_C_H____T_E_S_T____73()’:
/home/capsel/libs/frozen/tests/test_unordered_map_str.cpp:74:59: error: variable ‘olaf0’ set but not used [-Werror=unused-but-set-variable]
   constexpr frozen::unordered_map<frozen::string, int, 2> olaf0 = {
                                                           ^~~~~
/home/capsel/libs/frozen/tests/test_unordered_map_str.cpp:78:59: error: variable ‘olaf1’ set but not used [-Werror=unused-but-set-variable]
   constexpr frozen::unordered_map<frozen::string, int, 6> olaf1 = {
                                                           ^~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [tests/CMakeFiles/frozen.tests.dir/build.make:167: tests/CMakeFiles/frozen.tests.dir/test_unordered_map_str.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1005: tests/CMakeFiles/frozen.tests.dir/all] Error 2
make: *** [Makefile:163: all] Error 2

```

--- a/tests/test_unordered_map_str.cpp
+++ b/tests/test_unordered_map_str.cpp
@@ -83,4 +83,6 @@ TEST_CASE("various frozen::unordered_map config", "[unordered_map]") {
       {"nice damn shit", 14},
       {"pouet", 11},
   };
+  (void)olaf0;
+  (void)olaf1;
 }

